documentation/site/pages/developers/tutorials/request.mdx

```diff
@@ -75,17 +194,45 @@ let program_url = client.upload_program(program).await?;
 ```
 </StripRustCodeComments>
 
-### 3. Upload the Input of Your Program to IPFS
+After which, you'd create a request with:
 
-:::tip[*Rather not have inputs publicly available?**]
-When working with trusted provers, you can store inputs in *Amazon S3* and restrict access with a bucket policy (plus an optional *KMS key policy* if you enable server-side encryption).
-Only provers whose IAM role appears in those policies can download the inputs.
+```rust
+let request = client.new_request()
+  .with_program_url(&program_url)
+  .with_input_url(&input_url);
```

Comment:
So, `with_input_url` does not exist right now. This could be added, or it could be accomplished with `.with_request_input(Url::parse("https://myguest.null/program")?)`